roles/test_operator/defaults/main.yml

```diff
@@ -100,36 +103,36 @@ cifmw_test_operator_tempest_config:
     name: "{{ test_operator_job_name }}"
     namespace: "{{ cifmw_test_operator_namespace }}"
   spec:
-    containerImage: "{{ cifmw_test_operator_tempest_image }}:{{ cifmw_test_operator_tempest_image_tag }}"
-    storageClass: "{{ cifmw_test_operator_storage_class }}"
-    privileged: "{{ cifmw_test_operator_privileged }}"
-    parallel: "{{ cifmw_test_operator_tempest_parallel | default(omit) }}"
-    SSHKeySecretName: "{{ cifmw_test_operator_tempest_ssh_key_secret_name | default(omit) }}"
-    configOverwrite: "{{ cifmw_test_operator_tempest_config_overwrite | default(omit) }}"
-    networkAttachments: "{{ cifmw_test_operator_tempest_network_attachments }}"
+    containerImage: "{{ _test_operator_tempest_image }}:{{ _test_operator_tempest_image_tag }}"
+    storageClass: "{{ _test_operator_storage_class }}"
+    privileged: "{{ _test_operator_privileged }}"
+    parallel: "{{ _test_operator_tempest_parallel | default(omit) }}"
+    SSHKeySecretName: "{{ _test_operator_tempest_ssh_key_secret_name | default(omit) }}"
+    configOverwrite: "{{ _test_operator_tempest_config_overwrite | default(omit) }}"
+    networkAttachments: "{{ _test_operator_tempest_network_attachments }}"
     tolerations: "{{ cifmw_test_operator_tolerations | default(omit) }}"
     nodeSelector: "{{ cifmw_test_operator_node_selector | default(omit) }}"
-    extraConfigmapsMounts: "{{ cifmw_test_operator_tempest_extra_configmaps_mounts | default(omit) }}"
+    extraConfigmapsMounts: "{{ _test_operator_tempest_extra_configmaps_mounts | default(omit) }}"
     tempestRun:
       includeList: |
-        {{ cifmw_test_operator_tempest_include_list | default('') }}
+        {{ _test_operator_tempest_include_list | default('') }}
       excludeList: |
-        {{ cifmw_test_operator_tempest_exclude_list | default('') }}
+        {{ _test_operator_tempest_exclude_list | default('') }}
       concurrency: "{{ cifmw_test_operator_concurrency }}"
```

Comment:
I really wanted to do it but ansible is not preserving the jinja type when doing so which causes it to treat it like a string and pass it with '' which causes the applying of the CR to fail. unfortunately the only way I found to get the type preserved is to change this behavior globally using  [DEFAULT_JINJA2_NATIVE](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-jinja2-native) which might break other things (I tried `| int` and other options but they didn't work). if you are familiar with a better way to preserve jinja types let me know. please alos refer this issue in ansible project: https://github.com/ansible/ansible/issues/56898

Reply:
I was thinking about this and I'm afraid it is something that we need to resolve. I think that `int` parameters are not the only ones that are going to suffer from this issue. I suspect `bool` will be an issue as well. I do not have a solution right now. I have to investigate.

Reply:
I am not familiar with a way to solve it on the ansible side without using the configuration var I shared, having that we require this patch I would like to find an agreeable solution as soon as possible that would allow merging this patch. IMO we should discuss this issue with a broader audience to decide how to move on with it in case we don't have a clear solution for it.

Reply:
Let's talk this through during the meeting. I think that this issue together with this one [1] are the two major bottlenecks of this PR. 

[1] https://github.com/openstack-k8s-operators/ci-framework/pull/2374/files#r1795197746